CMakeLists.txt

```diff
@@ -1134,7 +1134,9 @@ option(ZLIB_ENABLE_TESTS "Build test binaries" ON)
 if(ZLIB_ENABLE_TESTS)
     enable_testing()
 
-    add_subdirectory(test)
+    if("${CMAKE_VERSION}" VERSION_GREATER_EQUAL "3.12")
+        add_subdirectory(test)
+    endif()
     if(WITH_BENCHMARKS)
```

Comment:
The `WITH_BENCHMARKS` if statement should also be wrapped in the version check because it also uses `FetchContent` to retrieve `googlebenchmark` same as test does with `googletest`.

Reply:
Done.